.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,17 @@
+name: C/C++ CI
+
+on: [push]
+
+jobs:
+  build-ubuntu:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: configure
+      run: mkdir build && cd build && cmake -DCMAKE_CXX_FLAGS="-Werror" ..
```

Comment:
we already have `-Werror -Wall -Wextra` specified in the CMake itself: will this override them or be in addition to?

Also with a more modern `cmake` installation one can do `cmake -S . -B build ...` too . I think ubuntu latest probalby has that. In fairness I had to undo those changes from the `Makefile`as the system I was testing on had an old cmake...

Reply:
Good question and I fear it may be the former (override) which was not my intent when copying these steps from a comment in https://github.com/actions/starter-workflows/pull/47 .

Oh, very nice! CMake continues to impress me... for years (for open source hobby projects) I stubbornly chose Autotools for imagined portability advantages (with other people running my chess AI or GUI code) which seem to be a red herring (now if another developer wants to try supporting another platform, they can PR a test and maybe even a patch!).

Cheers!